Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -137,7 +154,50 @@ public IStringSanitizer GetStringSanitizer(int workspaceID)
 		{
 			throw new NotImplementedException();
 		}
-	}
-
 
+		private void InstallKeplerResourceFiles(List<string> keplerFiles)

```

Comment:
are you planning to add an integration test for this method?

Reply:
Not yet